src/main/java/de/qabel/desktop/ui/accounting/identitycontextmenu/IdentityContextMenuController.java

```diff
@@ -50,6 +66,16 @@
     @Override
     public void initialize(URL location, ResourceBundle resources) {
         resourceBundle = resources;
+        createButtonsGraphics();
+    }
+
+    private void createButtonsGraphics() {
+        editButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/pencil.png"))));
+        removeButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/delete.png"))));
+        exportButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/upload.png"))));
+        privateKeyButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/qrcode.png"))));
+        publicKeyQRButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/qrcode.png"))));
+        publicKeyEmailButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/email.png"))));
```

Comment:
maybe try to extract the ImageView as well, we currently dont know if its possible.
it may happen that the static imageviiew gets duplicated or multiple parents. 
reuse even the imageview containing the image would be even better but not madantory